athena-neptune/src/main/java/com/amazonaws/athena/connectors/neptune/NeptuneConnection.java

```diff
@@ -30,7 +30,7 @@
 {
     private static Cluster cluster = null;
 
-    NeptuneConnection(String neptuneEndpoint, String neptunePort, boolean enabledIAM)
+    public NeptuneConnection(String neptuneEndpoint, String neptunePort, boolean enabledIAM)
```

Comment:
Why public? seems package private is fine for how it is being used. please revert unless you need it to be literally public.

Reply:
Made this private